docs/tutorial/airbyte_config_persistence.md

```diff
@@ -0,0 +1,524 @@
+
+    docker run -it --rm --volume airbyte_workspace:/data busybox
+
+This will drop you into an `sh` shell to allow you to do what you want inside a BusyBox system from which we can browse the filesystem and accessing to logs files:
+
+    ls /data/5/0/
+
+Example Output:
+
+    catalog.json                  normalize                     tap_config.json
+    logs.log                      singer_rendered_catalog.json  target_config.json
+
+Or you can simply run:
+
+
+
+```bash
+docker run -it --rm --volume airbyte_workspace:/data busybox ls /data/4/0
+```
+
+Example Output:
+
+    catalog.json                 singer_rendered_catalog.json
+    logs.log                     tap_config.json
+    normalize                    target_config.json
+
+
+
+```bash
+docker run -it --rm --volume airbyte_workspace:/data busybox cat /data/5/0/catalog.json
+```
+
+Example Output:
+
+    {"streams":[{"stream":{"name":"exchange_rate","json_schema":{"type":"object","properties":{"CHF":{"type":"number"},"HRK":{"type":"number"},"date":{"type":"string"},"MXN":{"type":"number"},"ZAR":{"type":"number"},"INR":{"type":"number"},"CNY":{"type":"number"},"THB":{"type":"number"},"AUD":{"type":"number"},"ILS":{"type":"number"},"KRW":{"type":"number"},"JPY":{"type":"number"},"PLN":{"type":"number"},"GBP":{"type":"number"},"IDR":{"type":"number"},"HUF":{"type":"number"},"PHP":{"type":"number"},"TRY":{"type":"number"},"RUB":{"type":"number"},"HKD":{"type":"number"},"ISK":{"type":"number"},"EUR":{"type":"number"},"DKK":{"type":"number"},"CAD":{"type":"number"},"MYR":{"type":"number"},"USD":{"type":"number"},"BGN":{"type":"number"},"NOK":{"type":"number"},"RON":{"type":"number"},"SGD":{"type":"number"},"CZK":{"type":"number"},"SEK":{"type":"number"},"NZD":{"type":"number"},"BRL":{"type":"number"}}},"supported_sync_modes":["full_refresh"],"default_cursor_field":[]},"sync_mode":"full_refresh","cursor_field":[]}]}
+
+## Check local data folder
+
+Since the job completed successfully, a new file should be available in the special `/local/` directory in the container which is mounted from `/tmp/airbyte_local` on the host machine.
+
+
+```bash
+#!/usr/bin/env bash
+
+echo "In the container:"
+
+docker run -it --rm -v /tmp/airbyte_local:/local busybox find /local
+
+echo ""
+echo "On the host:"
+
+find /tmp/airbyte_local
+```
+
+Example Output:
+
+    In the container:
+    /local
+    /local/data
+    /local/data/exchange_rate_raw.csv
+    
+    On the host:
+    /tmp/airbyte_local
+    /tmp/airbyte_local/data
+    /tmp/airbyte_local/data/exchange_rate_raw.csv
+
+
+## Backup Exchange Rate Source and Destination configurations
+
+In the following steps, we will play with persistence of configurations so let's make a backup of our newly added connectors for now:
+
+
+```bash
+mkdir -p $TUTORIAL_DIR/exchange-rate-setup
+
+docker cp airbyte-server:data $TUTORIAL_DIR/exchange-rate-setup
```

Comment:
what do you think about adding this command to the developing locally section as well. i think this tutorial is great for someone who wants to go through the full flow. if you're just trying to find the command to pull logs it might be a little hard to find. wondering if just adding a super brief section in the developing locally section of the docs would make sense. wdyt?